server/src/main/scala/com/cloudera/livy/server/interactive/InteractiveSession.scala

```diff
@@ -202,6 +204,21 @@ class InteractiveSession(
     }
   }
 
+  private def findSparkRArchives(): Seq[String] = {
+    sys.env.get("SPARKR_ARCHIVES_PATH")
+      .map(_.split(",").toSeq)
+      .getOrElse {
+        sys.env.get("SPARK_HOME").map { case sparkHome =>
+          val rLibPath = Seq(sparkHome, "R", "lib").mkString(File.separator)
+          val rArchivesFile = new File(rLibPath, "sparkr.zip")
```

Comment:
So, there's a minor problem here, in that sparkr.zip does not exist before Spark 1.6. Spark 1.5 has those files exploded into a directory, which won't work here.

I think that's fine, but we need to document it for users who might really want to run Spark 1.5.